### ISSUE TITLE
Shortcut key to bring up pulldown menus

### ISSUE THREAD

**Issue by blindphoneman** [joncampbell123/dosbox-x#2566]
Hi:

I use DOSBox-X in conjunction with a Windows Screen reader.  It doesn't read much, but, it will read the pull down menus very well.

The problem is that I must click with the mouse the initial item, (main, drive, etc.)  One of my screen readers allows me to simulate a mouse click, but, the other one which plays nicer with DOSBox-X gets confused and doesn't route the mouse pointer where it should go.

I realize this is not something you can fix, since it is the screen reader developers who handle this aspect.

What I'm wondering is: Could a shortcut key be added which activates the menu.  Even a key to activate the "main" item would work, since, then, I can use the keyboard to navigate to all the other menus.

I'm also going to present this to the developers of the NVDA screen reader.  Maybe they can recommend a work-around.

Thanks!

**Comment by Wengier** [comment]
Thanks for the suggestion! I agree that adding a shortcut key to bring up the drop-down menu is likely a good idea. There are 3 types of menus in DOSBox-X for different platforms which may need to be handled separately.

**Comment by specious** [comment]
I’m using dosbox-x with framebuffer on Linux (no X11 and no mouse). There seems to be no way to bring up the pulldown menu.
